packages/semantic-dom-diff/README.md

```diff
@@ -6,21 +6,119 @@
 ```bash
 yarn add @open-wc/semantic-dom-diff --dev
 ```
+`semantic-dom-diff` exports a function which takes a string of HTML, and returns a string of HTML. It restructures given HTML string, returning it in a format which can be used for comparison:
+- whitespace and newlines are normalized
+- tags and attributes are printed on individual lines
+- comments, style and script tags are removed
+- additional tags and attributes can optionally be ignored
 
-## Basics
-
+## Basic usage
 ```javascript
-import { getSemanticDomDiff } from '@open-wc/semantic-dom-diff';
+import getDiffableHTML from '@open-wc/semantic-dom-diff';
 
-const leftTree = `
+const leftTree = getDiffableHTML(`
   <div>foo</div>
-`;
-const rightTree = `
+`);
+const rightTree = getDiffableHTML(`
   <div>bar</div>
-`;
+`);
+
+// use any string comparison tool, for example chai:
+expect(leftTree).to.equal(rightTree);
+```
+
+## Ignoring tags and attributes
+When working with libraries or custom elements there might be parts of the renderd HTML which is random or otherwise outside of your control. On those cases, you might want to ignore certain attributes or tags entirely. This is possible by passing an options object.
```

Comment:
```suggestion
When working with libraries or custom elements there might be parts of the rendered HTML which is random or otherwise outside of your control. In those cases, you might want to ignore certain attributes or tags entirely. This is possible by passing an options object.
```